lib/fastlane/actions/reset_simulator.rb

```diff
@@ -0,0 +1,65 @@
+module Fastlane
+  module Actions
+    module SharedValues
+      RESET_SIMULATOR_CUSTOM_VALUE = :RESET_SIMULATOR_CUSTOM_VALUE
+    end
+
+    class ResetSimulatorAction < Action
+      def self.run(params)
+        require 'json'
+
+        unless Actions.sh_no_action("xcrun simctl list > /dev/null 2>&1 && echo 1 ||  echo 0", log: false).chomp.to_i == 1
```

Comment:
@KrauseFx This is the best I could come up with so far.
Is there any better way to check if command exists in Fastlane or Ruby in general?

Also, should it just display the warning or raise an exception instead and break the whole lane?